.github/workflows/stack-deployment.yaml

```diff
@@ -11,43 +15,60 @@ jobs:
   deploy:
     name: Deploy Stack
     runs-on: ubuntu-latest
-    env:
-      AWS_REGION: ${{ secrets.AWS_REGION }}
     steps:
-      - name: Checkout repository
+      - name: Clone repository
         uses: actions/checkout@v4
 
-      - name: Setup AWS credentials
+      - name: Configure AWS CLI credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: ${{ secrets.AWS_CDK_ROLE_ARN }}
           role-session-name: js-assertion-cheat-sheet-stack-deployment
           aws-region: ${{ secrets.AWS_REGION }}
 
-      - name: Setup Node.js
+      - name: Setup NodeJS LTS
         uses: actions/setup-node@v4
         with:
           node-version: lts/*
 
-      - name: Install dependencies
+      - name: Install CDK dependencies
         working-directory: cdk
         run: npm install
 
-      - name: Deploy stack
+      - name: Deploy using CDK
         working-directory: cdk
         run: npm run deploy -- --require-approval never --outputs-file ./cdk-outputs.json
 
+      - name: Set CDK output
+        working-directory: cdk
+        run: echo "CDK_OUTPUT=$(cat cdk-outputs.json | jq '.JsAssertionCheatSheet')" >> $GITHUB_ENV
+
+      - name: Print CDK output
+        uses: WcAServices/markdown-template-action@v1
+        with:
+          template: |
+            # CDK output
+
+            ```json
+            $env.CDK_OUTPUT
```

Comment:
I am not sure about this syntax, but either of the following should work:

1. `$CDK_OUTPUT` (https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable)
2. `$env:CDK_OUTPUT` (in PowerShell https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell)

As we don't use PS, I'd suggest using the 1st, always, as usual.